packages/white-whale-std/src/pool_manager.rs

```diff
@@ -246,3 +239,14 @@ pub enum QueryMsg {
     #[returns(PairInfo)]
     Pair { pair_identifier: String },
 }
+
+/// The response for the `NativeTokenDecimals` query.
+#[cw_serde]
+pub struct NativeTokenDecimalsResponse {
```

Comment:
why do we need to return `denom` and `decimals`? we already know them as were passed in the query.